runtime/src/main/java/org/corfudb/protocols/wireprotocol/MultipleReadRequest.java

```diff
@@ -16,24 +14,26 @@
 @Data
```

Comment:
Would be also greate to use `@Getter` instead of `@Data`

Reply:
Done.